README.md

```diff
@@ -67,7 +69,27 @@ To create a component, create a vue file in `.theme/components` directory e.g.
 `.theme/components/BasicComponent.vue`. Then use this component in a page as
```

Comment:
dogru satira comment yazamiyorum ama ilk cumle `To create a component ...` e.g.'ye kadar `## Components` basliginda kalsin, kalani basic'e paraphrase edip kaydiralim